lib/potassium/assets/app/javascript/app.spec.js

```diff
@@ -0,0 +1,14 @@
+import { shallowMount } from '@vue/test-utils';
+import App from 'app';
+
+describe('App', () => {
```

Comment:
'describe' is not defined  no-undef

Reply:
Todos estos errores los resuelve https://github.com/platanus/la-guia/pull/123